src/app/systelab-login/form-recovery/form-recovery.component.html

```diff
@@ -1,6 +1,6 @@
 <input #inputUserName="ngModel" type="text" class="w-100 slab-form__input slab-form__input--topradius" [class.slab-form_input--error]="inputUserName.invalid && inputUserName.touched"
        id="inputUserName" [(ngModel)]="userName" placeholder="{{txtUsername}}"
-       name="inputUserName" (keyup.enter)="doRecovery()" required autocomplete="off" maxlength="20" minlength="6">
```

Comment:
Can we set the maxlength as a configurable parameter?

Reply:
Added a new parameter